src/main/kotlin/org/jetbrains/research/testspark/tools/evosuite/generation/EvoSuiteProcessManager.kt

```diff
@@ -84,7 +84,8 @@ class EvoSuiteProcessManager(
             if (ToolUtils.isProcessStopped(errorMonitor, indicator)) return null
 
             val regex = Regex("version \"(.*?)\"")
-            val version = regex.find(CommandLineRunner.run(arrayListOf(evoSuiteSettingsState.javaPath, "-version")))
+            val versionCommandResult = CommandLineRunner.run(arrayListOf(evoSuiteSettingsState.javaPath, "-version"))
+            val version = regex.find(versionCommandResult.executionMessage)
```

Comment:
How about logging the `versionCommandResult`'s exit code and message?
it'll become easier to trace issues with the set `java` filepath when using EvoSuite.